frontend/encore/installation.rst

```diff
@@ -154,10 +155,8 @@ Next, open the new ``assets/app.js`` file which contains some JavaScript code
     // any CSS you import will output into a single css file (app.css in this case)
     import './styles/app.css';
 
-    // Need jQuery? Install it with "yarn add jquery"(or "npm install jquery"), then uncomment to import it.
-    // import $ from 'jquery';
-
-    console.log('Hello Webpack Encore! Edit me in assets/app.js');
+    // start the Stimulus application
+    import './bootstrap';
```

Comment:
small comment, related to https://github.com/symfony/ux/issues/210
can this file be renamed as ./stimulus-bootstrap for example?